base/libgit2/libgit2.jl

```diff
@@ -603,8 +605,14 @@ function __init__()
 
     err = ccall((:git_libgit2_init, :libgit2), Cint, ())
     err > 0 || throw(ErrorException("error initializing LibGit2 module"))
+    REFCOUNT[] += 1
```

Comment:
I've changed these to be atomic counters: can you have a quick look over it as I'm not really familiar with them.